### DISCUSSION TITLE
This element is not visible

### DISCUSSION THREAD

**Discussion by lucas-foliveira** [cypress-io/cypress#16448] [Questions and Help]
From time to time I get this message, even though Cypress can deal with the element. But other times, like when I tried to deal with radio buttons, it didn't work because of that and the buttons weren't checked. 

![Screen Shot 2021-05-11 at 09 54 42](https://user-images.githubusercontent.com/64436995/117819805-4a5a1200-b240-11eb-9a03-67087371db0f.png)

Is this a bug or should it be something I might be missing?

**Comment by sandra-ouadghiri** [comment]
Hi,

I don't think you share enough element for us to say (here for ex the error message & DOM extract...).
But here some ideas.
- look at the DOM for answers. It can be a parent element hiding it, even if visually transparent.
- do you change the execution resolution?
I will bet on 'NO' for both but double-triple check. Is it really always visible (at the DOM level) you-bet-on-your-heart?
This said
- If you had a cy.wait(5000) before does it helps? Do you have that with every element or not at all?
If yes, then maybe your page takes time to paint some elements and increasing the get timeout (locally, or globally) may help.

If nothing helps, either share more info, or speak to your front dev for a debug sessions?
